dev/tools/create_overlays.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# TODO:
+#
+# - Check if the branch already exists in the remote => checkout
+# - Better error handling
+# - Just checkout, don't build
+# - Rebase functionality
+#
+
+set -x
+set -e
+set -o pipefail
+
+# setup_contrib_git("_build_ci/fiat", "https://github.com/ejgallego/fiat-core.git")
+setup_contrib_git() {
+
+    local _DIR=$1
+    local _GITURL=$2
+
+    ( cd $_DIR && git checkout -b $OVERLAY_BRANCH && git remote add $DEVELOPER_NAME $_GITURL )
+
+}
+
+if [ $# -lt 3 ]; then
+  echo "usage: $0 github_username pr_number contrib1 ... contribN"
+  exit 1
+fi
+
+set +x
+. dev/ci/ci-basic-overlay.sh
+set -x
+
+DEVELOPER_NAME=$1
+shift
+PR_NUMBER=$1
+shift
+OVERLAY_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+OVERLAY_FILE=$(mktemp overlay-XXXX)
+
+# Create the overlay file
+
+echo "if [ \"\$CI_PULL_REQUEST\" = \"$PR_NUMBER\" ] || [ \"\$CI_BRANCH\" = \"$OVERLAY_BRANCH\" ]; then " > $OVERLAY_FILE
+echo >> $OVERLAY_FILE
+
+# We first try to build the contribs
+while test $# -gt 0
+do
+    _CONTRIB_NAME=$1
+    _CONTRIB_GITURL=${_CONTRIB_NAME}_CI_GITURL
+    _CONTRIB_GITURL=${!_CONTRIB_GITURL}
+    echo "Processing Contrib $_CONTRIB_NAME"
+
+    # check _CONTRIB_GIT exists and it is of the from github...
+
+    _CONTRIB_DIR=_build_ci/$_CONTRIB_NAME
+
+    _CONTRIB_GITSUFFIX=$(echo $_CONTRIB_GITURL | sed 's/https:\/\/github.com\/[^\/]*\///')
```

Comment:
```suggestion
    _CONTRIB_GITSUFFIX=${_CONTRIB_GITURL#https://github.com/*/}
```

Reply:
Done.

Reply:
Note this doesn't seem to work so I reverted for now.

Reply:
Can you explain more how it doesn't work? When I do
~~~bash
$ _foo=https://github.com/one/two
$ echo ${_foo#https://github.com/*/}
~~~
I get `two`.